src/components/button/code/views/buttonView.jsx

```diff
@@ -3,6 +3,12 @@
 var React = require('react');
 
 module.exports = React.createClass({
+
+  propTypes: {
+    purpose: React.PropTypes.oneOf(['primary', 'secondary', 'success', 'warning', 'danger', 'info']),
+    size: React.PropTypes.oneOf(['small', 'large', 'extra-large'])
```

Comment:
@jodiedoubleday I'm noting this for future reference, but if we use these enums elsewhere in the CSS conventions established by Bootstrap we may wish to pull these out into a central JS file so they can be reused and edited from a single place. Not an issue for merge.